SwiftAppTemplate/SwiftAppTemplate/APIClient.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  APIClient.swift
+//  SwiftAppTemplate
+//
+//  Created by Catriona Scott on 5/1/19.
+//  Copyright © 2019 Catriona Scott. All rights reserved.
+//
+
+import Alamofire
+
+// TODO: Make this class implement the ConnectionTokenProvider protocol
```

Comment:
Not sure we need to provide this much guidance – I think it'd be better to keep this SDK-agnostic (so you could use it for either stripe-terminal-ios or stripe-ios)

Reply:
Ok Cool, yah i was not totally sure so I added them but I will remove all my terminal specific TODOs 😁